src/foam/u2/wizard/data/PurgeCacheSaver.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.data',
+  name: 'PurgeCacheSaver',
+  extends: 'foam.u2.wizard.data.ProxySaver',
+
+  documentation: `
+    A proxy-saver that clears cache on a specified DAO.
```

Comment:
Please add additional documentation to explain when and why you would want to do this.